ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/NodeCve/AffectedNodesTable.tsx

```diff
@@ -33,9 +53,9 @@ export const affectedNodeFragment = gql`
         cluster {
             name
         }
-        nodeComponents {
+        nodeComponents(query: $query) {
             ...NodeComponentFragment
-            nodeVulnerabilities {
+            nodeVulnerabilities(query: $query) {
                 vulnerabilityId: id
                 cve
                 severity
```

Comment:
Noticed this mistake in the query - we want to filter the component and vuln lists by the current CVE and query. Without this param we would get the entire component list for a node.